### DISCUSSION TITLE
BlueIris basics?

### DISCUSSION THREAD

**Discussion by recoveringlawyer** [mrlt8/docker-wyze-bridge#1065] [Q&A] [answered]
I have Windows Docker Desktop up and  mrlt8/wyze-bridge:latest running. I can see snapshots of my cameras at localhost:5000. But no matter what inputs, incantations, or swear words I sling at BlueIris, I cannot get any of the cameras to give me anything but "No Signal" and either 1. "Failed to connect", 2. "Error 400 Bad Request", or 3. "Error 404 RTSP not found" when I try to add them to BlueIris. Can anyone direct me to a step-by-step for bridging to BlueIris - or any other help? I am a total noob on Github and Docker and am not a Linux programmer (though at least I do know how to use CMD), so I may need some handholding! Thanks in advance!

**Comment by mrlt8** [comment, marked as answer]
Can you view the RTSP stream in something like VLC? 
You can get the RTSP url by visiting the Web-UI and selecting Streams > RTSP for a specific camera.

**Comment by recoveringlawyer** [comment]
It’s really hard to believe it was this simple:  prompted by your message, I opened localhost:5000, selected a Wyze Cam Pan V3, obtained the RTSP stream and fed it to VLC. Worked fine. I then went to BlueIris to add a camera configuration page, inserted the URL, set it to RTSP://, but then, in a momentary lapse of attention, entered OK instead of putting in my Wyze credentials as I had been doing. The result - it works perfectly with the default user/password! 
I applaud your excellent work on this project. Like you said, “It Just Works”!! Thank you!

**Comment by recoveringlawyer** [comment]
It’s really hard to believe it was this simple:  prompted by your message, I opened localhost:5000, selected a Wyze Cam Pan V3, obtained the RTSP stream and fed it to VLC. Worked fine. I then went to BlueIris to add a camera configuration page, inserted the URL, set it to RTSP://, but then, in a momentary lapse of attention, entered OK instead of putting in my Wyze credentials as I had been doing. The result - it works perfectly with the default user/password!

I applaud your excellent work on this project. Like you said, “It Just Works”!! Thank you!



On Dec 23, 2023, at 11:31 PM, mrlt8 ***@***.***> wrote:



Can you view the RTSP stream in something like VLC?
You can get the RTSP url by visiting the Web-UI and selecting Streams > RTSP for a specific camera.

**Comment by vincom** [comment]
for users who had this issue with blue iris, this is a screenshot of my config. ports are the defaults from bridge
user or pw are not required, i dont know why


![blueirisdocker](https://github.com/mrlt8/docker-wyze-bridge/assets/2191155/04770eb9-67d6-47ae-ba50-0ee4a37cb602)

**Reply by shawnhaywood** [reply]
Was it necessary to open any ports on the wyze-bridge vm?
